<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>OptAppearanceMisc</class>
 <widget class="QWidget" name="OptAppearanceMisc" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OptAppearanceMiscUI</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox4" >
     <property name="title" >
      <string>Headings</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="ck_newHeadings" >
        <property name="text" >
         <string>Slim group headings</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="ck_outlineHeadings" >
        <property name="text" >
         <string>Outline headings</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_opacity" >
     <property name="title" >
      <string>Opacity</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="QSlider" name="sl_chatdlgop" >
        <property name="minimum" >
         <number>10</number>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="lb_chatdlgop_val" >
        <property name="minimumSize" >
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>100</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSlider" name="sl_rosterop" >
        <property name="minimum" >
         <number>10</number>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="lb_rosterop" >
        <property name="text" >
         <string>Roster opacity:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="lb_rosterop_val" >
        <property name="minimumSize" >
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>100</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lb_chatdlgop" >
        <property name="text" >
         <string>Chat dialog opacity:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>ck_outlineHeadings</tabstop>
  <tabstop>ck_newHeadings</tabstop>
  <tabstop>sl_rosterop</tabstop>
  <tabstop>sl_chatdlgop</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>sl_rosterop</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lb_rosterop_val</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>208</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel" >
     <x>320</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sl_chatdlgop</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lb_chatdlgop_val</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>208</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel" >
     <x>320</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
